.gitignore

```diff
@@ -1,16 +1,31 @@
-.DS_Store
-*.swp
-*.swo
-*.orig
-.idea
-tmp/
-ext/
-node_modules/
-coverage/
-.grunt/
-npm-debug.log
-bower_components/
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
 .grunt
-reports
-jsdoc
-test/src/
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# Commenting this out is preferred by some people, see
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+bower_components
+coverage
+tmp
+
+# Users Environment Variables
+.lock-wscript
```

Comment:
This is the boilerplate.  We'll want that DS_Store back at least...  jsdoc,,  not sure what else might be necessary